src/odemis/driver/technolution.py

```diff
@@ -987,17 +987,25 @@ def getYAcqSetpoints(self):
 
         return setpoints.tolist()
 
-    def getCalibrationSetpoints(self, total_line_scan_time):
+    def getCalibrationSetpoints(self, total_line_scan_time, sine_in_x=True, sine_in_y=False):
         """
         Calculate the setpoints for the descanner during calibration mode. The setpoints resemble a sine shape
-        in x and a flat line at zero in y.
+        on one axis and a flat line on the other or a sine in both directions.
 
         :param total_line_scan_time: (float) Total line scanning time in seconds including
-                                     overscanned pixels and flyback time. TODO do we need the flyback?
+                                     overscanned pixels and flyback time.
+        :param sine_in_x: (bool) If True, the setpoints in the x-direction will follow a sine wave.
+                                 if False, the setpoints in the x-direction will be a flat line.
+        :param sine_in_y: (bool) If True, the setpoints in the y-direction will follow a sine wave.
+                                 if False, the setpoints in the y-direction will be a flat line.
         :return:
                 x_setpoints (list of ints): The calibration setpoints in x direction in bits.
                 y_setpoints (list of ints): The calibration setpoints in y direction in bits.
         """
+        if not sine_in_x and not sine_in_y:
```

Comment:
```suggestion
        if not sine_in_x or not sine_in_y:
```

Reply:
It should only raise an error if both are False

Reply:
Right, my bad